src/glue/jobs/json_to_parquet.py

```diff
@@ -322,7 +326,8 @@ def write_table_to_s3(
             frame = dynamic_frame,
             connection_type = "s3",
             connection_options = {
-                "path": s3_write_path
+                "path": s3_write_path,
+                "partitionKeys": ["cohort"]
```

Comment:
Do you envision us adding more partitions in the future?  If so, does it make sense to add them sooner than later?

Reply:
I'm not convinced that partitions are particularly useful right now. It's a way to keep cohort data separate from one another, but I'm not convinced that there's a lot of I/O performance benefits right now. Adult data is like 99% of our total data for one. Digital health also seems to be working with the entire dataset for their processing, so partitions probably aren't getting used.